backend/prisma/schema.prisma

```diff
@@ -96,6 +96,20 @@ model ReverseShare {
   creator   User   @relation(fields: [creatorId], references: [id], onDelete: Cascade)
 
   shares Share[]
+  sharesOptions ReverseShareOptions?
+}
+
+model ReverseShareOptions {
+  id        String   @id @default(uuid())
+  share    ReverseShare @relation(fields: [shareId], references: [id], onDelete: Cascade)
+  shareId  String @unique
+
+  easyMode Boolean @default(false)
```

Comment:
Can't we remove easyMode? If I understand it right easyMode is the same as setting all other properties to false.

But I like the easyMode checkbox in the frontend, then a user doesn't have to click all other checkboxes to disable or enable all options.

Reply:
Well yes but if we add others parameters in the future it would be easier to keep an EasyMode no ?

Reply:
Okay makes sense, I just thought it could lead to more code e.g if you want to check whether you want to display the description input field you have to check `easyMode` and `descriptionEnabled`.  But if you think it's easier  for the future it's totally fine for me.

Reply:
Yes you're right, but I think adding easyMode really makes sense because we then understand that this won't show anything.